graphblas/io.py

```diff
@@ -482,6 +536,40 @@ def indices(self):
     return ret
 
 
+def to_pydata_sparse(A, format="coo"):
```

Comment:
Should we add `fill_value=None` to pass to the `sparse` array constructor?

Reply:
Not sure about this one.  Because of the intermediate `scipy` conversion (which doesn't accept `fill_value`), I don't know how this would look like.

Reply:
I think `fill_value` would be passed to `s.asformat(format, fill_value=fill_value)` at the very end.

Reply:
Hmm, `pydata.sparse` doesn't have that `kwarg`? https://sparse.pydata.org/en/stable/generated/sparse.SparseArray.asformat.html

Reply:
Oh, my bad, you're right. I meant e.g.:
```python
s = COO.from_scipy_sparse(B, fill_value=fill_value)
```
I think all `sparse` constructors accept `fill_value` (`.asformat` methods being the exception). Us doing this will reinforce the data model used by `sparse`.

Reply:
Nope, that's not an option either...  https://sparse.pydata.org/en/stable/generated/sparse.COO.from_scipy_sparse.html
The converter doesn't accept any arguments other than a scipy sparse matrix.

That's also why I had the deleted paragraph about missing/zero treatment being similar to `scipy.sparse` as in this case it would inherit whatever treatment `scipy.sparse` applied (though you are correct that's not in general the case, so that paragraph was deleted).

Reply:
heh, double oops. Right, it doesn't look like `sparse` handles `fill_value` at all with `scipy.sparse` objects. Okay, then no need to add it.